src/.vuepress/markdown/link.js

```diff
@@ -0,0 +1,100 @@
+// markdown-it plugin for:
+// 1. adding target="_blank" to external links
+// 2. converting internal links to <router-link>
+
+const url = require('url')
+
+const indexRE = /(^|.*\/)(index|readme).md(#?.*)$/i
+
+module.exports = (configuration) => {
+  return (md) => {
+    let hasOpenRouterLink = false
+    let hasOpenExternalLink = false
+
+    md.renderer.rules.link_open = (tokens, idx, options, env, self) => {
+    const { base } = configuration;
+    const startsWithBaseRE = new RegExp(`^${base}.*`)
+    const internalRE = /(\/|\.md|\.html)(#.*)?$/
```

Comment:
```suggestion
    const internalRE = /(\/|\.md|\.html)(#.+)?$/
```

because I don't think this should allow to match when there is only `/some/link/#` but shouldn't this match also when there is only something like this `#http` ?

Reply:
`/some/link/#` is definitely allowed